cmd/configschema/comments_test.go

```diff
@@ -18,12 +18,20 @@ import (
 	"reflect"
 	"testing"
 
+	"github.com/google/uuid"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestFieldComments(t *testing.T) {
 	v := reflect.ValueOf(testStruct{})
-	comments := commentsForStruct(v, testDR())
+	comments, _ := commentsForStruct(v, testDR())
```

Comment:
nit
```suggestion
	comments, err := commentsForStruct(v, testDR())
	assert.NoError(t, err)
```

Reply:
gotcha w/ the change, quick question though does "nit" stand or not in test?